library/swift/src/RequestFilterCallbacks.swift

```diff
@@ -1,5 +1,6 @@
 import Foundation
 
+/// Interface for RequestFilters to access FilterManager state.
```

Comment:
```suggestion
/// Interface for RequestFilters to access FilterManager state.
```

Might want to remove this (same in Kotlin) for now since we haven't yet confirmed if `FilterManager` will be a class or where that logic will live